src/Components/ProposalCard/ProposalCard.styled.ts

```diff
@@ -0,0 +1,134 @@
+import styled from 'styled-components';
+import { Box } from 'Components/Primitives/Layout';
+import { Heading } from 'old-components/Guilds/common/Typography';
+
+export const CardWrapper = styled(Box)`

```

Comment:
We need to base these components on a generic Card component at some point. (We have one in src/old-components/Guilds/common/Card/Card.tsx, just not too sure if its good enough to be used here.)

Reply:
Agree, I didn't add these components. Just create a proposalcard.styled and cut all the styled components created inside proposalCard file to split concerns.I think it would be great to close this pr first.

Reply:
I will add a todo